ma/api/category.py

```diff
@@ -39,12 +39,14 @@ def delete(self, category_id):
         is_deleted = self.magento.catalog_category.delete(category_id)
         return is_deleted
 
-    def disable(self, category_id):
+    def update_active_status(self, category_id, is_active=True):
         # magento requires sorting values to be passed
-        is_disabled = self.magento.catalog_category.update(
+        is_updated = self.magento.catalog_category.update(
             category_id,
-            {'is_active': '0', 'available_sort_by': 'position', 'default_sort_by': 'position'})
-        return is_disabled
+            { 'is_active': '1' if is_active else '0',
```

Comment:
That's fine, if you think the readability trumps the convention for this
particular thing (as opposed to every other occurrence). As long as it's
thoughtful and intentional.



Dustin

On Fri, May 13, 2016 at 4:40 PM, Diego Terzano <notifications@github.com>
wrote:

> In ma/api/category.py
> <https://github.com/CoffeeForThinkers/MagentoAPI/pull/2#discussion_r63247915>
> :
>
> >              category_id,
> > -            {'is_active': '0', 'available_sort_by': 'position', 'default_sort_by': 'position'})
> > -        return is_disabled
> > +            { 'is_active': '1' if is_active else '0',
>
> ok, will do. I thought about doing that but since Magento sometimes uses 0
> for True, I wanted it to be VERY explicit and readable.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/CoffeeForThinkers/MagentoAPI/pull/2/files/7bb2b461d8e77954a6eacddfe18a443d1670f44c#r63247915>
>